students/slupek_bartosz/lesson_02_flow_control/Lost_cards.py

```diff
@@ -0,0 +1,7 @@
+n = int(input())
+sum_cards = 0
+for i in range(1, n + 1):
```

Comment:
Jeśli chcemy przyśpieszyć to można również skorzystać ze wzoru na sumę szeregu:
https://pl.wikipedia.org/wiki/Szereg_1_%2B_2_%2B_3_%2B_4_%2B_%E2%80%A6